hedera-mirror-rosetta/app/domain/types/address_book_entry_test.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestToRosettaPeers(t *testing.T) {
+	exampleAccount, _ := AccountFromString("0.0.0")
+	var testData = []*AddressBookEntry{
+		{exampleAccount, nil},
+	}
+	exampleEntries := &AddressBookEntries{
+		testData,
+	}
+	result := exampleEntries.ToRosettaPeers()
+
+	assert.Len(t, result, 1)
+	assert.Equal(t, testData[0].PeerId.String(), result[0].PeerID)
+	assert.Equal(t, testData[0].Metadata, result[0].Metadata)
```

Comment:
I think that it would be better to assert that the whole object equals:
```Go
expected := []*rTypes.Peer{
		{PeerID: exampleAccount.String(), Metadata: nil},
}
assert.Equal(t, expected, result)
```